src/ceph-disk/ceph_disk/main.py

```diff
@@ -4524,9 +4531,35 @@ def list_devices():
     return devices
 
 
+def list_zfs():
+    try:
+        out, err, ret = command(
+            [   'zfs',
+                'list',
+                '-o', 'name,mountpoint'
+            ]
+           )
+    except subprocess.CalledProcessError as e:
+        LOG.info('zfs list -o name,mountpoint '
+           'fails.\n (Error: %s)' % e)
+        return 1
```

Comment:
we should re-raise the exception. so `main_catch()` can catch it, or, user can get more clue pass `--verbose` to ceph-disk at seeing the error, which will print out the backtrace in any exception is raised.